phpstan/action.yml

```diff
@@ -0,0 +1,37 @@
+name: "PHPStan"
+description: "phpstan"
+author: "shopware AG"
+branding:
+  color: "blue"
+  icon: "download"
+
+inputs:
+  extensionName:
+    description: "Your extension name"
+    required: true
+  cacheDir:
+    description: PHPStan cache dir
+    default: ${{ github.workspace }}/var/phpstan
+
+runs:
+  using: "composite"
+  steps:
+    - name: Create phpstan.neon
+      working-directory: custom/plugins/${{ inputs.extensionName }}
+      shell: bash
+      run: |
+        printf "parameters:\n  tmpDir: ${{ inputs.cacheDir }}\nincludes:\n  - phpstan.neon.dist" > phpstan.neon
```

Comment:
Aren't we ignoring any future additions/changes in `phpstan.neon` this way?. Maybe configuring the cache can be achieved using `sed` or something, just replacing the `tmpDir`.

Reply:
If there's no phpstan.neon, it should work. I'll check our plugins, if all of them use phpstan.neon.dist, it should work. We could also add check that the file does not exist yet

Reply:
> We could also add check that the file does not exist yet

Sounds good 👍 As long as there's no proper way to decode/encode `neon` using a standalone binary or standard CLI tooling, it's also the only option I suppose.